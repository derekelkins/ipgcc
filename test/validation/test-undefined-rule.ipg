S 
  -> A
     for i = 0 to 3 do B[i, i+1]
     { a = A(1).y }
     repeat D.x
     repeat E.x until F
     { s = D.START }
     { e = E.END };

/* Expands to the following hence the "extra" errors.
S
  -> A@0[0, EOI]
     for i = 0 to 3 do B@0[i, i + 1]
     { a = A@0(1).y }
     repeat D@0[D@0.END, EOI].x starting on [B@0.END, EOI]
     repeat E@0[E@0.END, EOI].x starting on [D@0.END, EOI] until F@0
     { s = D@0.START }
     { e = E@0.END };
 */
