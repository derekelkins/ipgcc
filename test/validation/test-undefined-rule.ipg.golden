Rule A in rule S is undefined
Rule B in rule S is undefined
Rule A in rule S is undefined
Rule D in rule S is undefined
Rule D in rule S is undefined
Rule B in rule S is undefined
Rule E in rule S is undefined
Rule F in rule S is undefined
Rule E in rule S is undefined
Rule D in rule S is undefined
Rule D in rule S is undefined
Rule E in rule S is undefined